lpr.UnitTests/PackageTest.cs

```diff
@@ -90,46 +95,74 @@ public async void ArchivePackageAsync_Throws_ApiExecption_No_Pacakage()
         public async void CreatePackageAsync_Throws_ApiException_String_Empty()
         {
             Package testPackage = LoopFaker.Package();
+            Account testAccount = LoopFaker.Account();
 
             testPackage.Name = "";
 
-            PackageDataMock.Setup(d => d.CreatePackageAsync(testPackage))
+            PackageDataMock.Setup(d => d.CreatePackageAsync(null, testAccount.Id, testPackage))
                 .ReturnsAsync(testPackage);
+
+            AccountDataMock.Setup(d => d.GetAccountById(testAccount.Id))
+                .ReturnsAsync(testAccount);
             
-            IPackageService packageService = new PackageService(PackageDataMock.Object);
+            IPackageService packageService = new PackageService(PackageDataMock.Object, this.AccountDataMock.Object);
 
-            await Assert.ThrowsAsync<ApiException>(() => packageService.CreatePackageAsync(testPackage));
+            await Assert.ThrowsAsync<ApiException>(() => packageService.CreatePackageAsync(null, testAccount.Id, testPackage));
         }
 
         [Fact]
         public async void CreatePackageAsync_Throws_ApiException_Invalid_Character()
         {
             Package testPackage = LoopFaker.Package();
+            Account testAccount = LoopFaker.Account();
 
             testPackage.Name = "TestPackage#";
 
-            PackageDataMock.Setup(d => d.CreatePackageAsync(testPackage))
+            PackageDataMock.Setup(d => d.CreatePackageAsync(null, testAccount.Id, testPackage))
                 .ReturnsAsync(testPackage);
 
-            IPackageService packageService = new PackageService(PackageDataMock.Object);
+            AccountDataMock.Setup(d => d.GetAccountById(testAccount.Id))
+                .ReturnsAsync(testAccount);
 
-            await Assert.ThrowsAsync<ApiException>(() => packageService.CreatePackageAsync(testPackage));
+            IPackageService packageService = new PackageService(PackageDataMock.Object, this.AccountDataMock.Object);
+
+            await Assert.ThrowsAsync<ApiException>(() => packageService.CreatePackageAsync(null, testAccount.Id, testPackage));
         }
 
 
         [Fact]
         public async void CreatePackageAsync_Throws_ApiException_String_TooShort()
         {
             Package testPackage = LoopFaker.Package();
+            Account testAccount = LoopFaker.Account();
+
+            testPackage.Name = "test";
```

Comment:
Just a question, Why the name change?

Reply:
For this method (CreatePackageAsync_Throws_ApiException_String_TooShort), the change is needed.
But the method (CreatePackageAsync_Throws_ApiException_Account_Doesnt_Exist) was a copy from this method and I forgot to delete it there.